bike-erp/backend/src/main/services/triggerService/TriggerService.ts

```diff
@@ -0,0 +1,23 @@
+import { TriggersDao } from "../../dao/TriggersDao";
+
+const triggersDao =  new TriggersDao();
+  
+  export class TriggerService {
+
+    // Retrieve all triggers from stored in the table
+    public getAllTriggers = () => {
+      return triggersDao.fetchAllTriggers();
+    };
+
+    // Retrive trigger state by ID
```

Comment:
Can some comments be added about the valid trigger ids for people using the API?